admin_manual/configuration_server/occ_command.rst

```diff
@@ -43,8 +65,8 @@ Query your ownCloud server status::
    Array
    (
     [installed] => true
-    [version] => 7.0.4.2
-    [versionstring] => 7.0.4
+    [version] => 8.1.0.3
```

Comment:
the output here is different:
```
$ ./occ status
 - installed: true
 - version: 8.1.0.4
 - versionstring: 8.1 alpha 3
 - edition: 
```

We also added json/pretty json support to it:
```
$ ./occ status --output=json
{"installed":true,"version":"8.1.0.4","versionstring":"8.1 alpha 3","edition":""}

$ ./occ status --output=json_pretty
{
    "installed": true,
    "version": "8.1.0.4",
    "versionstring": "8.1 alpha 3",
    "edition": ""
}
```
which allows external tools to parse this information in a better way. This applies to all kind of lists we have:
status, check, app:list, encryption:list-modules, encryption:status ( https://github.com/owncloud/core/pull/16044 )